Dockerfile

```diff
@@ -8,11 +8,29 @@ RUN apt update && \
 COPY files/usr/local/bin/entrypoint /usr/local/bin/entrypoint
 
 # Install peertube-runner
-RUN npm install -g @peertube/peertube-runner@${PEERTUBE_RUNNER_VERSION:-"0.0.9"}
+RUN npm install -g @peertube/peertube-runner@${PEERTUBE_RUNNER_VERSION:-"0.0.17"}
 
 # Un-privileged user running the application
 ARG DOCKER_USER
 USER ${DOCKER_USER}
 
 ENTRYPOINT [ "entrypoint" ]
-CMD [ "peertube-runner", "server" ]
\ No newline at end of file
+CMD [ "peertube-runner", "server" ]
+
+FROM runner AS whisper_ctranslate2
+
+USER root:root
+
+RUN apt-get update && \
+    apt install -y python3-pip && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt install python3-pip -y
+
+RUN pip3 install whisper-ctranslate2==0.4.5 --break-system-packages
+
+# Un-privileged user running the application
+ARG DOCKER_USER
+USER ${DOCKER_USER}
+
+RUN whisper-ctranslate2 --model tiny unknown
```

Comment:
IMO you should not do that. This will make the image bigger for nothing and embed an image that can be out of date in few months if we don't create a new tag.
IMO it should be the user responsibility to determine when, which one and how the model is downloaded.

Reply:
You're right, good catch :facepalm:

Reply:
I remove this part.